src/applications/personalization/profile/selectors.js

```diff
@@ -80,5 +80,12 @@ export const militaryInformationLoadError = state => {
   return state.vaProfile?.militaryInformation?.serviceHistory?.error;
 };
 
-export const showNotificationSettings = state =>
-  toggleValues(state)[FEATURE_FLAG_NAMES.profileNotificationSettings];
+export const showNotificationSettings = state => {
+  const LSProfileNotificationSetting = localStorage.getItem(
+    'PROFILE_NOTIFICATION_SETTINGS', // 0 or 1
+  );
+  const FFProfileNotificationSettings = toggleValues(state)[
+    FEATURE_FLAG_NAMES.profileNotificationSettings
+  ];
+  return !!(FFProfileNotificationSettings || LSProfileNotificationSetting);
```

Comment:
We're going to need to be more precise than this since the value of `PROFILE_NOTIFICATION_SETTINGS` is a string and would always be truthy if its value was '0' or '1' or anything else.

And we want to make sure that the local storage setting overrides the feature flag if the local storage setting is set to '0' or '1':

we want this selector to return true if:
- the feature flag is on
- the LS flag is '1'

we want this selector to return false if:
- the feature flag is off
- the LS flag is '0' (even if the feature flag is on)

Reply:
good point sorry about that. Will get that done tomorrow morning